File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverExplore/Address.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Address {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("house")
    @Expose
    private String house;
    @SerializedName("street")
    @Expose
    private String street;
    @SerializedName("district")
    @Expose
    private String district;
    @SerializedName("city")
    @Expose
    private String city;
    @SerializedName("county")
    @Expose
    private String county;
    @SerializedName("country")
    @Expose
    private String country;
    @SerializedName("countryCode")
    @Expose
    private String countryCode;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getHouse() {
        return house;
    }

    public void setHouse(String house) {
        this.house = house;
    }

    public String getStreet() {
        return street;
    }

    public void setStreet(String street) {
        this.street = street;
    }

    public String getDistrict() {
        return district;
    }

    public void setDistrict(String district) {
        this.district = district;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getCounty() {
        return county;
    }

    public void setCounty(String county) {
        this.county = county;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getCountryCode() {
        return countryCode;
    }

    public void setCountryCode(String countryCode) {
        this.countryCode = countryCode;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/Address.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import com.google.gson.annotations.Expose

class Address {
    @SerializedName("text")
    @Expose
    private var text: String? = null

    @SerializedName("house")
    @Expose
    private var house: String? = null

    @SerializedName("street")
    @Expose
    private var street: String? = null

    @SerializedName("postalCode")
    @Expose
    private var postalCode: String? = null

    @SerializedName("district")
    @Expose
    private var district: String? = null

    @SerializedName("city")
    @Expose
    private var city: String? = null

    @SerializedName("county")
    @Expose
    private var county: String? = null

    @SerializedName("state")
    @Expose
    private var state: String? = null

    @SerializedName("country")
    @Expose
    private var country: String? = null

    @SerializedName("countryCode")
    @Expose
    private var countryCode: String? = null
    fun getText(): String? {
        return text
    }

    fun setText(text: String?) {
        this.text = text
    }

    fun getHouse(): String? {
        return house
    }

    fun setHouse(house: String?) {
        this.house = house
    }

    fun getStreet(): String? {
        return street
    }

    fun setStreet(street: String?) {
        this.street = street
    }

    fun getPostalCode(): String? {
        return postalCode
    }

    fun setPostalCode(postalCode: String?) {
        this.postalCode = postalCode
    }

    fun getDistrict(): String? {
        return district
    }

    fun setDistrict(district: String?) {
        this.district = district
    }

    fun getCity(): String? {
        return city
    }

    fun setCity(city: String?) {
        this.city = city
    }

    fun getCounty(): String? {
        return county
    }

    fun setCounty(county: String?) {
        this.county = county
    }

    fun getState(): String? {
        return state
    }

    fun setState(state: String?) {
        this.state = state
    }

    fun getCountry(): String? {
        return country
    }

    fun setCountry(country: String?) {
        this.country = country
    }

    fun getCountryCode(): String? {
        return countryCode
    }

    fun setCountryCode(countryCode: String?) {
        this.countryCode = countryCode
    }
}